src/Shared/Recruit.Shared.Web/ViewModels/VacancySummaryViewModel.cs

```diff
@@ -23,7 +23,7 @@ public class VacancySummaryViewModel
 
         public bool HasVacancyReference => VacancyReference.HasValue;
         public bool HasNoVacancyReference => !HasVacancyReference;
-        public bool IsApplicationsVacancy => (Status== VacancyStatus.Live || Status ==VacancyStatus.Closed) 
+        public bool IsApplicationsVacancy => (Status== VacancyStatus.Live || Status == VacancyStatus.Closed) 
```

Comment:
CanShowApplicationsOnVacancy instead of IsApplicationsVacancy?

Reply:
I don't agree with your suggestions. I am calling it `CanShowVacancyApplicationsCount`

Reply:
If that's how you want to be 😉  ... sounds good 👍